### PR TITLE
Seed Data Cleanup

### PR DESCRIPTION
**Pull request by sarvaiyanidhi**
This PR address below changes in the seed file
- Defined fixed set of users for better flow
- Mapped users to defined workspace "Caley" and personal workspace
- Defined two specific teams and mapped different set of users to each of them
- Added multiple email servers to users with combination of Gmail and Outlook
- Defined user without team mapped only to personal workspace

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -10,80 +10,167 @@
...
-  puts "Loading seed data for user: #{user.email}"
+  })
+end
+puts "Created User for Caley workspace - #{user.email}"
 
-  # Create a workspace for each user
-  workspace = Workspace.create!(
-    name: "#{Faker::Company.name} Workspace",
+# Create Caley workspace
+name = "Caley"
```
Comment (wassson) [inline db/seeds.rb:29]: Small detail, but would you make the workspace names and team names constants and put them at the top of the file? I'd like to make it clear in the future that these shouldn't be changed
Ex: `WORKSPACE_NAME = "Caley".freeze`

File: db/seeds.rb [hunk 2]
```diff
@@ -10,80 +10,167 @@
...
-  )
+  })
+end
+puts "Created Workspace - #{caley_workspace.name}"
 
-  # Create a team within the workspace
-  team = Team.create!(
-    name: "#{Faker::Team.name} Team",
+# Create iOS and Ruby on Rails teams for Caley workspace
+ios_team_name = "iOS"
```
Comment (wassson) [inline db/seeds.rb:40]: Same comment as above!

File: db/seeds.rb [hunk 3]
```diff
@@ -10,80 +10,167 @@
...
-      user: user,
-      email_server: email_server,
-      snippet: Faker::Lorem.sentence,
-      summary: Faker::Lorem.sentence
+    workspace_id: caley_workspace.id
+  })
+end
+puts "Created Team - #{ios_team.name}"
+
+ror_team_name = "Ruby on Rails"
```
Comment (wassson) [inline db/seeds.rb:50]: Same comment!

**Review by wassson — CHANGES_REQUESTED** [review]
Thank you for the hard work! I apologize that this ended up needing two PRs. Would you mind making the final small change? I'll make sure to get this through asap

**Comment by sarvaiyanidhi** [comment]
Thank you for the feedback @wassson .. Have pushed requested changes along with some other minor changes to check  same users getting added again in workspace and team user mapping table incase if we execute seed script again in already loaded data. Not very clean solution as it executes query again to check user but thought as it is small set of data so should be fine for it in case of optimizing it further.

**Review by wassson — APPROVED** [review]
Thank you!

**Pull request merged**